### PR TITLE
Recipe for excitingtools python package

### PR DESCRIPTION
**Pull request by AlexBuccheri**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AlexBuccheri** [comment]
@conda-forge-admin, please ping conda-forge/help-python for review

File: recipes/excitingtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=35.0.2
+    - setuptools-scm >=2.0.0,<3
```
Comment (beenje) [inline recipes/excitingtools/meta.yaml:21]: Any reason for pinning `setuptools-scm` to such an old version (2018)?
First version available on conda-forge is 3.5.0. Current version is 7.1.0.

Comment (AlexBuccheri) [reply recipes/excitingtools/meta.yaml:21]: An oversight on my part when moving from `setup.py` to `pyproject.toml` (I copy-pasted `[build-system] `from an example).

Updated to `setuptools-scm >= 3.5.0` in the recipe and the `excitingtools` project.

File: recipes/excitingtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=35.0.2
+    - setuptools-scm >=3.5.0
+    - pip
+  run:
+    - python >=3.6
+    - wheel >=0.35.0
```
Comment (dopplershift) [inline recipes/excitingtools/meta.yaml:25]: I won't hold up for this since it matches upstream but I'm 99% sure you're not relying on wheel at run-time.

**Review by dopplershift — APPROVED** [review]

File: recipes/excitingtools/meta.yaml (hunk 2 above)
Comment (AlexBuccheri) [reply recipes/excitingtools/meta.yaml:25]: Thanks, very appreciated. I've taken another look and both the `setuptools-scm` pinning and addition of `wheels` as a run time dependency were a result of auto-generating the meta yaml with [grayskull](https://github.com/conda/grayskull). Admittedly I should have noticed, but I wasn't looking too hard 😅

**Pull request merged**